tests/ecosystem/upgrade/test_resources.py

```diff
@@ -13,7 +11,6 @@
 log = logging.getLogger(__name__)
 
 
-@post_upgrade
```

Comment:
@petr-balogh Cool, makes sense. Should we proceed with the change in https://github.com/red-hat-storage/ocs-ci/pull/2505?

Reply:
I have the PR  #2505 ready, but not sure if it will be accepted. So maybe I will just take the one specific commit out of this PR, or you can just use that approach in this PR and I will rebase my PR on top of yours. WDYT?

Reply:
Will use your approach in this PR so https://github.com/red-hat-storage/ocs-ci/pull/2505 scope will be for the inteligent ceph health

Reply:
done

Reply:
@ebenahar this test_storage_pods_running I think is not IO related and as I mentioned I think should not be disabled here. `@post_upgrade` should not be removed I think.

Reply:
@ebondare have you missed this comment?

Reply:
sorry, I did!
Will re-enable this test